cpp/src/DataStorm/SessionI.cpp

```diff
@@ -1222,15 +1202,16 @@ SubscriberSessionI::getTopics(const string& name) const
 }
 
 void
-SubscriberSessionI::s(int64_t topicId, int64_t elementId, DataSample s, const Ice::Current& current)
+SubscriberSessionI::s(int64_t topicId, int64_t elementId, DataSample dataSample, const Ice::Current& current)
 {
```

Comment:
I kept the same logic, just renames parameters, and loop variables to make things more readable.